cmd/transferer/config.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+type PMMConfig struct {
+	PmmURL             string
+	ClickHouseURL      string
+	VictoriaMetricsURL string
+}
+
+func getPMMConfig(pmmLink, vmLink, chLink string) (PMMConfig, error) {
+	var result PMMConfig
+	pmmURL, err := url.Parse(pmmLink)
+	if err != nil {
+		return result, fmt.Errorf("failed to parse pmm_url: %s", err)
+	}
+
+	result.PmmURL = pmmLink
+	if vmLink != "" {
+		result.VictoriaMetricsURL = vmLink
+	} else {
+		result.VictoriaMetricsURL = modifyURL(*pmmURL, "/prometheus", pmmURL.User, "", "")
+	}
+	if chLink != "" {
+		result.ClickHouseURL = chLink
+	} else {
+		result.ClickHouseURL = modifyURL(*pmmURL, "", nil, "9000", "database=pmm")
+	}
+
+	return result, nil
```

Comment:
Could be simplified, but test it first :)

```suggestion
	pmmURL, err := url.Parse(pmmLink)
	if err != nil {
		return result, fmt.Errorf("failed to parse pmm_url: %s", err)
	}
	
	var conf = PMMConfig{
		PmmURL: pmmLink,
		VictoriaMetricsURL: vmLink,
		ClickHouseURL: chLink,
	}

	if conf.VictoriaMetricsURL == "" {
		conf.VictoriaMetricsURL = modifyURL(*pmmURL, "/prometheus", pmmURL.User, "", "")
	}
	
	if conf.ClickHouseURL == "" {
		conf.ClickHouseURL = modifyURL(*pmmURL, "", nil, "9000", "database=pmm")
	}

	return conf, nil
```